packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -236,6 +237,7 @@ class RenderEditable extends RenderBox with RelayoutWhenSystemFontsChangeMixin {
          textDirection: textDirection,
          textScaleFactor: textScaleFactor,
          locale: locale,
+         ellipsis: text?.style?.overflow == TextOverflow.ellipsis ? kDefaultEllipsis : null,
```

Comment:
@chunhtai this is the ony real change to consider in this PR (the other ones was just renaming the ellipsis property to be able to reuse it).

I idea for the test is to build a `SelectableText` -> find the `RenderEditable` -> then verify that is a the ellipsis passed accordingly. I believe that would make more sense for this PR as the overflow test already exist and as you said we can trust the current implementation that the ellipsis will be shown proeprly.

Please let me know your thoughts.